### PR TITLE
Feature/add world

### PR DESCRIPTION
**Pull request by Yadunund**
This PR allows users to launch the office world in Gazebo.

### CODE REVIEW

File: demos/launch/common.launch.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          args="$(var config_file)"/>
+  </group>
+
+  <!-- Visualizer -->
+  <include file="$(find-pkg-share visualizer)/visualizer.xml">
+    <arg name="map_name" value="L1"/>
+    <arg name="viz_config_file" value ="$(find-pkg-share demos)/include/office.rviz"/>"/>
+  </include>
+
+</launch>
```
Comment (aaronchongth) [inline demos/launch/common.launch.xml]: super small nitpick

File: demos/launch/office.launch.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <env name="GAZEBO_MODEL_DATABASE_URI" value="" />
+    </executable>
+    <executable cmd="gzclient --verbose $(var world_path)" output="both">
+      <env name="GAZEBO_MODEL_PATH" value="$(var model_path)" />
+      <env name="GAZEBO_RESOURCE_PATH" value="$(var resource_path)" />
+      <env name="GAZEBO_PLUGIN_PATH" value="$(var plugin_path)" />
+    </executable>
+  </group>
+
+  </launch>
```
Comment (aaronchongth) [inline demos/launch/office.launch.xml]: smae

File: rmf_demo_assets/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+project(rmf_demo_assets)
+
+find_package(ament_cmake REQUIRED)
+
+install(
+  DIRECTORY models
+  DESTINATION share/${PROJECT_NAME}
+)
+
+ament_package()
```
Comment (aaronchongth) [inline rmf_demo_assets/CMakeLists.txt]: new EOL

File: rmf_demo_maps/CMakeLists.txt [hunk 4]
```diff
@@ -10,3 +10,50 @@ install(DIRECTORY
...
+  add_custom_target(generate_${output_world_name}_nav_graphs ALL
+    DEPENDS ${output_nav_graphs_phony}
+  )
+
+  install(
+    DIRECTORY ${output_dir}
+    DESTINATION share/${PROJECT_NAME}/maps
+  )
+
+endforeach()
```
Comment (aaronchongth) [inline rmf_demo_maps/CMakeLists.txt]: newline EOF

Comment (Yadunund) [reply rmf_demo_maps/CMakeLists.txt]: Thanks for the feedback.

**Pull request merged**